dangerzone/container_utils.py

```diff
@@ -1,16 +1,17 @@
+import functools
 import logging
 import os
 import platform
 import shutil
 import subprocess
 from pathlib import Path
-from typing import List, Optional, Tuple
+from typing import IO, Callable, Iterable, List, Optional, Tuple
 
 from . import errors
 from .settings import Settings
 from .util import get_resource_path, get_subprocess_startupinfo
 
-CONTAINER_NAME = "dangerzone.rocks/dangerzone"
+OLD_CONTAINER_NAME = "dangerzone.rocks/dangerzone"
```

Comment:
Quick question, do you plan to use the old container name somewhere? Asking because I don't see anyone using this constant.

Reply:
We have to keep it for a few releases, to be able to remove the old image after an upgrade to the dangerzone version that will introduce indep. container updates.

It's not used right now, but it's on my todo :-)

Reply:
I expected as much, thanks!